contents/projectManagement/documentation.md

```diff
@@ -0,0 +1,158 @@
+* markup: use formatting such as colors and italics and structural elements like admonition blocks to highlight important points
+* realistic examples
+
+Futhermore, you should always:
+
+* [test](http://idratherbewriting.com/2015/07/07/testing-your-instructions/) your instructions
+* find *someone else* to test your instructions
+* [revise, revise and revise!](https://jacobian.org/writing/editors/)
+
+[This](http://www.bluemangolearning.com/software-documentation/) provides useful general tips for writing documentation and 
+[this guide](https://jacobian.org/writing/technical-style/) provides more tips on writing well. 
+
+#### README
+
+A README serves as an introduction to your project, and is often the first thing people see when encountering your project.
+A good README should convince users to use your product or contribute to your code, by providing enough information to make it easy for them to get started.
+For a good guide on how to write a README, check out [this guide](http://www.writethedocs.org/guide/writing/beginners-guide-to-docs/).
+Some suggest that READMEs should be written before you start coding, as it gives you an overview of your project and what you need to do.
+This is known as [Readme Driven Development](http://tom.preston-werner.com/2010/08/23/readme-driven-development.html).
+
+#### User Guide
+
+A user guide teaches users how to use your software.
+User guides can include quickstart guides, reference materials, tutorials, cookbooks/recipes and more.
+[Here](https://jacobian.org/writing/what-to-write/) is a useful guide on how to tackle different types of user documentation.
+[This](http://stevelosh.com/blog/2013/09/teach-dont-tell/) points out the problems with common approaches to documentation and builds on the previous article.
+
+#### Developer Guide
+
+Developer guides teach new developers how to contribute to the code base.
+They should give an overview of the code and an explanation of the design, as well as include instructions for setting up, making changes and deploying.
+See [this](https://www.petrikainulainen.net/software-development/processes/writing-just-enough-documentation/) to learn more about what to include in a developer guide.
+
+### Code Documentation
+
+Code docummentation refers to documentation that is found within the source code.
+All the following types of code documentation are important and complement one another.
+Note that such documentation should always be updated together with the code.
+
+#### Comments
+
+Comments are a well-known form of documentation.
+They [complement the code](http://softwareengineering.stackexchange.com/a/285789) and are intended for developers who use and work on the code.
+They include variable, method and class comments, as well as inline comments. 
```

Comment:
Consider rephrasing to:

They include **API documentation** for variables, methods, classes, and packages, as well as **inline comments**.